src-tauri/src/main.rs

```diff
@@ -74,3 +87,42 @@ async fn async_process_model(
 
     Ok(())
 }
+
+#[tauri::command]
+fn run_articulation_point(nodes: Vec<NeighborInfo>) -> String {
```

Comment:
Can you move these functions to a new file? They probably shouldn't be in main.rs

Reply:
done; moved to aux_functions